Sources/ActionSheet/ActionSheetCell.swift

```diff
@@ -56,16 +107,35 @@ class ActionSheetCell: UICollectionViewCell {
         name.setContentHuggingPriority(.defaultLow, for: .horizontal)
         return name
     }()
-
-    let checkmark: UILabel = {
-        let checkmark = UILabel()
-        checkmark.text = "✓"
-        checkmark.font = UIFont.systemFont(ofSize: 18)
-        checkmark.textColor = PresentationTheme.current.colors.orangeUI
-        checkmark.translatesAutoresizingMaskIntoConstraints = false
-        checkmark.isHidden = true
-        return checkmark
+    
+    private var accessoryTypeImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.contentMode = .scaleAspectFit
+        imageView.backgroundColor = .none
+        imageView.tintColor = PresentationTheme.current.colors.cellDetailTextColor
+        imageView.image = .checkmark
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        return imageView
     }()
+    
+    var accessoryType: ActionSheetCellAccessoryType {
+        didSet {
+            switch accessoryType {
+            case .checkmark:
+                accessoryTypeImageView.image = .checkmark
+                accessoryTypeImageView.isHidden = !isSelected
+            case .disclosureChevron:
+                accessoryTypeImageView.image = .disclosureChevron
+            case .custom(let image):
+                accessoryTypeImageView.image = image.withRenderingMode(.alwaysTemplate)
+            case .none:
+                accessoryTypeImageView.image = nil
+                accessoryTypeImageView.isHidden = true
+            default:
+                assertionFailure("Unhandled ActionSheetCellAccessoryType")
+            }
```

Comment:
Are these cells being reused? if so you could end up with a hidden imageview if you don't set that correctly in the other cases